src/Kunstmaan/NodeBundle/Helper/Services/PageCreatorService.php

```diff
@@ -58,54 +93,58 @@ public function setUserEntityClass($userEntityClass)
      *
      * @api
      */
-    public function setContainer(ContainerInterface $container = null)
+    public function setContainer(ContainerInterface $container)
     {
-        $this->setEntityManager($container->get('doctrine.orm.entity_manager'));
-        $this->setACLPermissionCreatorService($container->get('kunstmaan_node.acl_permission_creator_service'));
+        @trigger_error(
+            'Container injection is deprecated in KunstmaanNodeBundle 5.1 and will be removed in KunstmaanNodeBundle 6.0.',
+            E_USER_DEPRECATED
+        );
+        $this->setEntityManager($container->get(EntityManagerInterface::class));
+        $this->setAclPermissionCreatorService($container->get(ACLPermissionCreatorService::class));
         $this->setUserEntityClass($container->getParameter('fos_user.model.user.class'));
     }
 
     /**
      * @param HasNodeInterface $pageTypeInstance The page.
      * @param array            $translations     Containing arrays. Sample:
-     * [
-     *  [   "language" => "nl",
-     *      "callback" => function($page, $translation) {
-     *          $translation->setTitle('NL titel');
-     *      }
-     *  ],
-     *  [   "language" => "fr",
-     *      "callback" => function($page, $translation) {
-     *          $translation->setTitle('FR titel');
-     *      }
-     *  ]
-     * ]
-     * Perhaps it's cleaner when you create one array and append another array for each language.
+     *                                           [
+     *                                           [   "language" => "nl",
+     *                                           "callback" => function($page, $translation) {
+     *                                           $translation->setTitle('NL titel');
+     *                                           }
+     *                                           ],
+     *                                           [   "language" => "fr",
+     *                                           "callback" => function($page, $translation) {
+     *                                           $translation->setTitle('FR titel');
+     *                                           }
+     *                                           ]
+     *                                           ]
+     *                                           Perhaps it's cleaner when you create one array and append another array for each language.
```

Comment:
This should be reverted. But I think if you move this example to the phpdoc description part, it won't get reformatted